packages/loot-core/src/server/main.js

```diff
@@ -2170,7 +2170,7 @@ export async function initApp(version, isDev, socketName) {
   }
 }
 
-export async function init({ budgetId, config }) {
+export async function init(config) {
```

Comment:
You'll still want `budgetId` because you only need to set up and download the file once. In future runs you can pass in `budgetId` directly... but I see your point about separating them actually. We need the API to be something like:

```
api.init(config);
api.downloadFile({ id: ... })
```

And `downloadFile` will not do anything if the file is already downloaded. We need it separate because you want to run the same script regardless of if it's already downloaded or not. So pulling out `budgetId` makes sense!